R/database.R

```diff
@@ -2734,16 +2697,19 @@ get_manta_sv_by_samples = function(these_samples_metadata,
 #'
 #' @examples
 #' my_svs = get_manta_sv_by_sample(this_sample_id = "00-14595_tumorD",
-#'                              these_samples_metadata = get_gambl_metadata() %>% dplyr::filter(sample_id == "00-14595_tumorD"))
+#'                                 these_samples_metadata = get_gambl_metadata())
 #'
 get_manta_sv_by_sample = function(this_sample_id,
                                   these_samples_metadata,
-                                  force_lift = TRUE,
+                                  force_lift = FALSE,
                                   min_vaf = 0.1,
                                   min_score = 40,
                                   pass = TRUE,
-                                  with_chr_prefix = FALSE,
                                   projection = "grch37"){
+
```

Comment:
So this message runs a bunch of times even if this function is called the normal way? Or is the liftover done elsewhere in that case?

Reply:
This message only gets printed if the user calls this function directly and with `force_lift = TRUE` (default is FALSE). When this function is internally called by `get_manta_sv_by_samples` this is hard coded to be FALSE, so that all the lifting takes place within that function, on one data frame, instead of running `liftover_bedpe` on all individual flat-files. If that make sense?

Reply:
Yep. Thanks